src/components/Popups/ReplyField/ReplyField.tsx

```diff
@@ -124,12 +167,58 @@ export default class ReplyField extends React.Component<Props, State> {
         setCursorPosition(editorState.getSelection().getFocusOffset());
     };
 
+    stopDefaultEvent = (e: React.KeyboardEvent): void => {
```

Comment:
We should try to avoid abbreviating parameter names. Does this need to be a keyboard event?